### DISCUSSION TITLE
Merging fix for pixi-projection

### DISCUSSION THREAD

**Discussion by kozickikarol** [pixijs/pixijs#8049] [Q&A] [answered]
Hey guys, is there anyone who manages the pixi-projection repository and can see https://github.com/pixijs/pixi-projection/pull/101.

**Comment by kozickikarol** [comment, marked as answer]
Is here anyone in charge who can check it?

**Reply by bigtimebuddy** [reply]
I merged it, sorry. Just waiting on @ivanpopelyshev to grant me publishing access.

**Reply by bigtimebuddy** [reply]
Done! v0.4.4 is up and contains the fix.
